ui/src/css/components/_RepoHeader.scss

```diff
@@ -0,0 +1,27 @@
+#repoheader {
+  border-bottom: 1px solid #dadce0;
+  overflow: auto;
+}
+
+#repoheader-backbutton {
+  float: left;
+  margin: 18px 20px 0 16px;
+}
+
+#repoheader-info {
+  float: left;
+}
+
+#repoheader p {
+  margin: 0;
+}
+
+#repoheader p.mdc-typography--subtitle1 {
+  color: #212121;
+  margin: 3px 0 8px;
```

Comment:
This might change -- I'll ask Travis about the sizes and spacing along with the other stuff (using the subtitle class for the header feels a little odd, but it's the only thing that seems to be the right text size).